source/Server.Contracts/Accounts/IAccountTypeProvider.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using FluentValidation;
+using Octopus.Server.Extensibility.Extensions.Mappings;
+
+namespace Sashimi.Server.Contracts.Accounts
+{
+    public interface IAccountTypeProvider: IContributeMappings, IServiceMessageHandler
```

Comment:
Some of these signatures are different from Server, do the same refactoring on the Server side before deleting the Server version